omod/src/main/resources/apps/ugandaemr_reports_app.json

```diff
@@ -606,6 +606,15 @@
         "url": "reportingui/runReport.page?reportDefinition=1381401f-1dd2-11b2-96d6-3d6bf54c9095",
         "order": 150,
         "requiredPrivilege": "App: ugandaemrreports.patientRegimenChangeHistory"
+      },
+      {
+        "id": "Back To Care Services",
+        "extensionPointId": "org.openmrs.module.ugandaemr.reports.overview",
+        "type": "link",
+        "label": "Back To Care Services",
```

Comment:
Rename this to Back To Care Services Report or  Back To Care Services Report Register